camunda-formio-tasklist-vue/src/components/TaskList.vue

```diff
@@ -209,33 +212,34 @@
                   </div>
                 </div>
                 <div class="cft-user-details" v-else>
-                  <b-tooltip target="setAssignee" triggers="hover">
-                    Click to change <b>assignee</b>
-                  </b-tooltip>
-                  <span id="setAssignee">
+                  <span
+                    id="setAssignee"
+                    v-b-tooltip.hover
+                    title="Click to change Assignee"
+                  >
                     <i class="fa fa-user cft-person-fill" />
                     <span class="cft-user-span" @click="toggleassignee">
                       {{ task.assignee }}
                     </span>
                   </span>
-                  <b-tooltip target="resetAssignee" triggers="hover">
-                    Reset <b>Assignee</b>
-                  </b-tooltip>
-                  <span id="resetAssignee">
+                  <span
+                    id="resetAssignee"
+                    v-b-tooltip.hover
+                    title="Reset Assignee"
+                  >
                     <i class="fa fa-times ml-1" @click="onUnClaim" />
                   </span>
                 </div>
               </b-col>
-              <b-col v-else cols="12" md="2">
-                <div @click="onClaim">
+              <b-col
+                :class="task.assignee ? '' : 'd-flex align-items-end'"
+                v-else
+              >
+                <div @click="onClaim" v-b-tooltip.hover.left title="Claim task">
                   <span id="claimAssignee">
-                    <i class="fa fa-user" />
-                    Claim
+                    <i class="fa fa-user" />Claim
```

Comment:
Would be good to give two spaces after <i> tag to give some gap after icon and Claim